doc/conf.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+"""Sphinx configurations to build package documentation."""
+
+from documenteer.sphinxconfig.stackconf import build_package_configs
+
+import lsst.validate.base
+
+
+_g = globals()
+_g.update(build_package_configs(
+    project_name='validate_base',
+    copyright='2017 Association of Universities for '
+              'Research in Astronomy, Inc.',
+    version=lsst.validate.metrics.__version__,
+    doxygen_xml_dirname=None))
+
+intersphinx_mapping['astropy'] = ('http://docs.astropy.org/en/stable', None)
+intersphinx_mapping['validate_base'] = ('https://validate-base.lsst.io', None)
```

Comment:
This one's legit; it lets us reference the `validate_base` user guide from a (future) `validate_metrics` user guide.